packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -190,9 +203,24 @@ export default {
             });
         }
         this.isIosBrowser = /(iPhone|iPad).*Safari/.test(navigator.userAgent);
+
+        if (this.shouldAbsolutePositionReopenLink) {
+            this.bodyObserver = new ResizeObserver(this.updateIsBodyHeightLessThanWindowHeight);
+            const bodyElement = document.documentElement || document.body;
+            this.bodyObserver.observe(bodyElement);
+        }
     },
 
     methods: {
+        /**
+         * Triggered by <body> ResizeObserver calculates if the body height less than the window
+         */
+        updateIsBodyHeightLessThanWindowHeight () {
+            const ropenElementHeight = this.$refs.reopenCookieBannerLink.$el.clientHeight || 0;
```

Comment:
Is the `r` at the start of the variable name supposed to be there?

Reply:
Ooops thanks